package/MDAnalysis/core/groups.py

```diff
@@ -2623,6 +2623,12 @@ def select_atoms(self, sel, *othersel, **selgroups):
         periodic : bool (optional)
           for geometric selections, whether to account for atoms in different
           periodic images when searching
+        atol : float, optional
```

Comment:
I can document this, but I've actually moved those keywords up into the function signature to avoid redundancy. I can't recall why we couldn't do this before, it may have been a 2.7 thing?